doc/src/user_guide.rst

```diff
@@ -287,6 +287,12 @@ Parameterises the solver with
 
     where
 
+    ``euler`` requires
```

Comment:
?

Reply:
Sorry left over from asking for the docs to be a seperate PR